### PR TITLE
Fixes a couple issues with extremely outdated copypaste coat code

### LINKED ISSUE #4621: (WEB REPORT BY: broccollin REMOTE: 198.245.63.50:7777) Lawyers Blue Suit Jacket Bugged
> Revision (Should be above if you're viewing this from ingame!)
> 
> General description of the issue
> The Lawyer blue suit can hold 2 small items, but the menu doesn't work correctly, it will stay up when you open another container. There are other issues with it too, like putting a beaker in the jacket will splash the content onto the jacket, and then put an empty beaker in the jacker.
> I like playing lawyer and storing things in the jacket is useful, mostly because no one knows you can
> 
> What you expected to happen
> 
> What actually happened
> 
> Steps to reproduce if possible

### PR DESCRIPTION
**Pull request by ghost**
Fixes #4621

### CODE REVIEW

**Comment by Exxion** [comment]
OK I know you're going for SPEED but maybe consider fixing this properly, like how storage accessories do it

**Comment by 9600bauds** [comment]
That's a good idea

**Comment by Exxion** [comment]
Hell, you could even just give them an unremovable storage accessory

**Comment by 9600bauds** [comment]
I had completely forgotten storage accessories were a thing, thank you

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -402,9 +402,9 @@
 		for(var/obj/item/weapon/storage/S in src.contents)	//Check for storage items
 			L += get_contents(S)
 		for(var/obj/item/clothing/suit/storage/S in src.contents)//Check for labcoats and jackets
-			L += get_contents(S)
+			L += get_contents(S.hold)
 		for(var/obj/item/clothing/accessory/storage/S in src.contents)//Check for holsters
-			L += get_contents(S)
+			L += get_contents(S.hold)
```
Comment (9600bauds) [inline code/modules/mob/living/living.dm:407]: Hey look, I even fixed a bug with storage accessories

Comment (Exxion) [reply code/modules/mob/living/living.dm:407]: I was going to suggest that you define `get_contents()` for those two things as `return get_contents(hold)`, but then I remembered that `get_contents()` is already such a tremendous clusterfuck that it doesn't really matter

<sub>I'm pretty sure I agreed at some point to unfuck it but oops</sub>

**Pull request merged**